### DISCUSSION TITLE
Can I change jellyfin docker settings to do not delete logs?

### DISCUSSION THREAD

**Discussion by bbyongi** [jellyfin/jellyfin#11104] [Q&A] [answered]
I run jellyfin server with docker-compose.yml

> version: "3"
> services:
>   jellyfin:
>     image: jellyfin/jellyfin:latest
>     container_name: jellyfin
>     restart: always
>     volumes:
>       - ./volume/config:/config
>       - ./volume/cache:/cache
>       - ./volume/video:/media
>     ports:
>      - 8096:8096
> 

However, the default jellyfin docker setting only saved logs for 3 days.
All logs older than 3 days are being deleted.

![1](https://github.com/jellyfin/jellyfin/assets/58283298/4967ee42-e5cf-42e0-8c43-0d74e96afb6e)

![2](https://github.com/jellyfin/jellyfin/assets/58283298/020d4c57-c571-4aa7-a504-228c20f36e7b)

I would like to keep the log for a month or more longger.
Can I change jellyfin docker settings to do not delete logs or a month?

If possible, please tell me how to change/setting it.

Thank you.

**Comment by TimGels** [comment, marked as answer]
Hi there!

could you try editing the logging.json file and setting the number on your installation to the value you desire? I dont know from the top of my head how you can access volumes, as you apparently use volumes to store the config. Otherwise you can probably edit it from inside the docker container.

It is the following file.
https://github.com/jellyfin/jellyfin/blob/83d2bc3f9f13c62f6d3ef16c4fe75f0f5a18110d/Jellyfin.Server/Resources/Configuration/logging.json#L26

https://jellyfin.org/docs/general/administration/configuration/#main-configuration

**Reply by bbyongi** [reply]
Thank you so much.
I will try to change it.

**Reply by bbyongi** [reply]
Thank you for your helping.
After changed 3 to 30 two day ago, I find 5 logs in my setting menu.
